### ISSUE TITLE
Demo page's labels aren't associated with input fields

### ISSUE THREAD

**Issue by cordeliadillon** [jxnblk/colorable#3]
This tool is great! However, I noticed one small accessibility issue - the labels for form fields on https://jxnblk.com/colorable/demos/text are missing `for` attributes, so they're not linking up properly with the input fields themselves.

**Comment by jxnblk** [comment]
Thanks! Sorry I missed that. I'll need to set up the React components to accept an id for each instance, but will update it soon.
